src/scripts/main.js

```diff
@@ -22,101 +23,170 @@ const chromeStorage = {
 };
 
 const bringDown = (function(){
-	
+
 	const bootTime = new Date();
-	let targetHour = 24;	
+	let targetHour = 24;
 	let restartTime;
 	let bringDownSet = false;
-	
+
 	function setTimeUntilBD(){
-		
+
 		if(bringDownSet){
 			console.warn("The bring-down has already been set. You cannot reset it. This system will restart in %d seconds", restartTime);
 			return;
 		}
-		
+
 		// If the targetHour has already passed in the day, the next opportunity is the next day
 		// So, here, figure out how long we have to wait until the same time tomorrow.
 		if(targetHour < bootTime.getHours()){
 			targetHour = 24 - (bootTime.getHours() - targetHour);
 		}
-		
+
 		const hoursUntilTarget = ((targetHour - bootTime.getHours()) * 60 * 60) * 1000;
 		const minutesFromNextHour = ((60 - bootTime.getMinutes()) * 60) * 1000;
-		
+
 		const millisecondsUntilEarliestPossibleRestart = hoursUntilTarget - minutesFromNextHour;
 		const giveALittle = (60 * 60 * 1000) * Math.random() | 0;
-		
+
 		const restartIn = millisecondsUntilEarliestPossibleRestart + giveALittle;
-		
+
 		setTimeout(function(){
-			
+
 			chrome.runtime.restart();
-					
+
 		}, restartIn);
-		
+
 		bringDownSet = true;
 		restartTime = Date.now() + restartIn;
-		
-		// The unix time that the Chrome OS device will be restarted at. 
+
+		// The unix time that the Chrome OS device will be restarted at.
 		return restartTime;
-		
+
 	}
-	
+
 	function getTimeUntilBD(){
-		
+
 		return restartTime;
-		
+
 	}
-	
+
 	return {
 		set : setTimeUntilBD,
 		get : getTimeUntilBD
 	};
-	
-})();
 
-const view = document.querySelector('webview');
+}());
+
+const views = [].slice.call(document.querySelectorAll('webview'));
```

Comment:
We use OBT (which converts ES6), could we not use Array.from instead? The effect is the same, but it's a little easier to read.